frontend/src/components/DocumentUnitAttachments.vue

```diff
@@ -125,20 +104,10 @@ async function upload(files: FileList) {
     }
   } finally {
     isLoading.value = false
-    emit("updateDocumentUnit")
-    if (!showExtraContentPanelRef.value && anySuccessful) {
-      toggleExtraContentPanel(true)
-    }
+    emit("attachmentsUploaded", anySuccessful)
   }
 }
 
-const toggleExtraContentPanel = (state?: boolean) => {
-  showExtraContentPanelRef.value = state || !showExtraContentPanelRef.value
-  pushQueryToRoute({
-    showAttachmentPanel: showExtraContentPanelRef.value.toString(),
-  })
-}
-
 function toggleDeleteModal() {
   showDeleteModal.value = !showDeleteModal.value
```

Comment:
We should move the checks whether the attachment index actually exists from the template to here. Otherwise, it might not be possible to close the dialog. Also, I think I prefer a open and close method instead of toggle, because the user operations are distinct. (close dialog button vs. delete file icon)

Reply:
very good point, let's fix it :)